doc_swift_ui_gallery/swift_ui_gallery/swift_ui_gallery/layouts/VStackExamples.swift

```diff
@@ -2,66 +2,201 @@ import SwiftUI
+
+struct VStackContentRightAlignedPage: View {
+    var body: some View {
+        NavigationStack {
+            VStack(alignment:.trailing) {
+                Text("First")
+                Text("Second")
+                Text("Third")
+            }
+        }.navigationTitle("Content Right Aligned")
+    }
+}
+
+struct VStackContentSpacingPage: View {
+    var body: some View {
+        NavigationStack {
+            VStack(alignment: .leading, spacing:100) {
+                Text("First")
+                Text("Second")
+                Text("Third")
+            }
+        }.navigationTitle("Content Spacing")
+    }
+}
+
+struct VStackExceedsAvailableSpacePage : View {
+    var body: some View {
+        NavigationStack {
+            VStack() {
+                Rectangle()
+                    .foregroundColor(Color.red)
+                    .frame(height: 200)
+                Rectangle()
+                    .foregroundColor(Color.green)
+                    .frame(height: 200)
+                Rectangle()
+                    .foregroundColor(Color.blue)
+                    .frame(height: 200)
+                Rectangle()
+                    .foregroundColor(Color.purple)
+                    .frame(height: 200)
+                Rectangle()
+                    .foregroundColor(Color.orange)
+                    .frame(height: 200)
+                Rectangle()
+                    .foregroundColor(Color.cyan)
+                    .frame(height: 200)
+                Rectangle()
+                    .foregroundColor(Color.yellow)
+                    .frame(height: 200)
+            }
+        }.navigationTitle("VStack Exceeds Space")
+    }
+}
+
+struct LazyVStackExceedsAvailableSpacePage: View {
+    var body: some View {
+        NavigationStack {
+            LazyVStack() {
+                Rectangle()
+                    .foregroundColor(Color.red)
+                    .frame(height: 200)
+                Rectangle()
+                    .foregroundColor(Color.green)
+                    .frame(height: 200)
+                Rectangle()
+                    .foregroundColor(Color.blue)
+                    .frame(height: 200)
+                Rectangle()
+                    .foregroundColor(Color.purple)
+                    .frame(height: 200)
+                Rectangle()
+                    .foregroundColor(Color.orange)
+                    .frame(height: 200)
+                Rectangle()
+                    .foregroundColor(Color.cyan)
+                    .frame(height: 200)
+                Rectangle()
+                    .foregroundColor(Color.yellow)
+                    .frame(height: 200)
+            }
+        }.navigationTitle("LazyVStack Exceeds Space")
     }
 }
 
 
 struct VStackTopAlignedPage_Previews: PreviewProvider {
     static var previews: some View {
-        VStackExceedsAvailableSpaceAlignedPage()
+        VStackTopAlignedPage()
+    }
+}
+
+struct VStackMiddleAlignedPage_Previews: PreviewProvider {
```

Comment:
Let's move each of these previews below their respective views.

Reply:
Done! Added previews just below the layouts.